solarwinds_apm.gemspec

```diff
@@ -45,14 +45,10 @@ Gem::Specification.new do |s|
 
   s.extensions = ['ext/oboe_metal/extconf.rb']
 
-  # OTEL dependencies
+  s.add_dependency('json')
```

Comment:
Got me thinking, how is this `json` gem used by our distro... not seeing anything outside of GHA workflows.

Reply:
It's probably from legacy code. I will remove it.